src/components/dotCMS/layout/LayoutGrid.tsx

```diff
@@ -48,7 +50,13 @@ export const LayoutGrid = () => {
                    * - We loop through the contentlets and render the contentlet
                    */
                   return (
-                    <Contentlet data={contentlet} key={contentlet.identifier} />
+                    <ContentletWrapper
```

Comment:
Missing container wrapper: https://github.com/dotCMS/dotcms-nextjs-example/blob/issue-21138-SPA-not-showing-container-wrapper-for-EMA/src/components/dotCMS/layout/Container.tsx#L57